wrappers/node/test/suite1/ariesvcx-proof.test.ts

```diff
@@ -8,9 +8,10 @@ import {
 } from 'helpers/entities'
 import { initVcxTestMode, shouldThrow } from 'helpers/utils'
 import { Connection, DisclosedProof, Proof, ProofState, StateType, VCXCode, VCXMock, VCXMockMessage } from 'src'
+import {PROTOCOL_TYPE_ARIES} from '../helpers/test-constants'
```

Comment:
I am so sorry for nitpicking, but please keep the bracketing consistent :) (in all the imports)